test/com/sun/jna/VarArgsTest.java

```diff
@@ -89,6 +90,12 @@ public void testStringVarArgs() {
                      lib.returnStringVarArgs("", args));
     }
 
+    public void testStringVarArgsFull() {
+        Object[] args = new Object[] { "Test" };
+        assertEquals("Did not return correct string", args[0],
+                     lib.returnStringVarArgs("", "Test"));
```

Comment:
I think you'd want to test passing _no_ arguments here.  Perhaps this "bug" is an issue with the client (java) code?